test/tests/ver-1kb/ver-1kb.i

```diff
@@ -0,0 +1,211 @@
+nb_segments = 20
+long_segment = '${units 1.25e-5 m}'
+long_total = '${units ${fparse nb_segments * long_segment} m}'
+simulation_time = '${units 10000 s}'
+T = '${units 500 K}'
+R = '${units 8.31446261815324 J/mol/K}' # ideal gas constant from PhysicalConstants.h
+Na = '${units 6.02214076e23 1/mol}' # Avogadro's number from PhysicalConstants.h
+initial_pressure_1 = '${units 1e5 Pa}'
+initial_pressure_2 = '${units 1e-10 Pa}'
+initial_concentration_1 = '${units ${fparse initial_pressure_1 / (R*T)} mol/m^3}'
+initial_concentration_2 = '${units ${fparse initial_pressure_2 / (R*T)} mol/m^3}'
+solubility = '${fparse 1.082e20/Na}' # Henry's law constant
+diffusivity = '${units ${fparse 4.31e-6 * exp(-2818/T)} m^2/s}'
```

Comment:
```suggestion
nb_segments_TMAP7 = 20
node_size_TMAP7 = ${units 1.25e-5 m}
long_total = ${fparse nb_segments_TMAP7 * node_size_TMAP7} # m
nb_segments_TMAP8 = 100
node_size_TMAP8 = ${fparse long_total / nb_segments_TMAP8} # m
simulation_time = ${units 10000 s}
temperature = ${units 500 K}
R = ${units 8.31446261815324 J/mol/K} # ideal gas constant from PhysicalConstants.h
initial_pressure_1 = ${units 1e5 Pa}
initial_pressure_2 = ${units 1e-10 Pa}
initial_concentration_1 = ${units ${fparse initial_pressure_1 / (R*temperature)} mol/m^3}
initial_concentration_2 = ${units ${fparse initial_pressure_2 / (R*temperature)} mol/m^3}
solubility = ${units ${fparse 1/(R*temperature)} mol/m^3/Pa} # Henry's law solubility
diffusivity = ${units ${fparse 4.31e-6 * exp(-2818/temperature)} m^2/s}
```
This is some clean up, but there is also two significant changes here: the solubility, and the mesh size. 

For the solubility, see general comment. 
For the mesh size, a finer mesh size will give you a higher accuracy for the flux of species across the interface. With a larger mesh, you might find that mass conservation is not as well respected as with a fine mesh. I set it to 100 here, but make sure to play around with different values to see how it affects the simulation results.